### DISCUSSION TITLE
3.6.0 Pre-release

### DISCUSSION THREAD

**Discussion by snowman2** [pyproj4/pyproj#1300] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/pyproj4/pyproj/releases/tag/3.6.0rc0'>3.6.0rc0</a>.</em>

Release Candidate 0 Status:

- [x] Linux ARM wheels (https://github.com/pyproj4/pyproj-wheels/pull/57)
- [x] Conda Forge (https://github.com/conda-forge/pyproj-feedstock/pull/137; https://github.com/conda-forge/pyproj-feedstock/pull/136)
- [x] Debian (@sebastic)
- [x] Fedora (@jdkloe)

Plan to release in a few days. Testing is always appreciated, but definitely not required.

**Comment by sebastic** [comment]
The Debian package built successfully with PROJ 9.1.1 from unstable:

 https://buildd.debian.org/status/package.php?p=python-pyproj&suite=experimental

**Reply by snowman2** [reply]
Thanks @sebastic :+1:

**Comment by jdkloe** [comment]
Also on Fedora Rawhide the version 3.6.0rc0 builds just fine.
See: https://koji.fedoraproject.org/koji/taskinfo?taskID=101958644
proj version was 9.2.1
python version was 3.11.3

**Reply by snowman2** [reply]
Thanks @jdkloe 👍
